lib/tether/tether.go

```diff
@@ -251,8 +251,18 @@ func (t *tether) setHostname() error {
 }
 
 func (t *tether) setNetworks() error {
+	var networks []*NetworkEndpoint
+	// prioritize internal networks
 	for _, v := range t.config.Networks {
-		if err := t.ops.Apply(v); err != nil {
+		if !v.Internal {
+			networks = append(networks, v)
+		} else {
+			networks = append([]*NetworkEndpoint{v}, networks...)
+		}
+	}
+
+	for _, network := range networks {
```

Comment:
Looking at this logic, it makes me think that you don't even need to build an extra array. Just go over local networks first and than over non local.